bumping head when on slope turns rabbit. fixes #68 and fixes #91.

colonelfazackerley opened the pull request.

andybalaam commented:
Lovely simple fix. Really rubs in the need for us to be able to test that all levels are soluble after behaviour changes (which I am working on).

I'd prefer to assert that the rabbits have turned around rather than they haven't not turned around. E.g. If you made a typo in the badLines string anywhere the test would pass even if the bug were not fixed. 

I'd mildly prefer the test to be in TestWalking but could be persuaded otherwise.

colonelfazackerley commented:
i updated the branch. 
- Another test in the other sense?
- I was thinking it might be nice to have a list of bug fix tests, but I am not all bothered. I moved it.

andybalaam commented:
Great, thanks.
